docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -3,9 +3,13 @@ title: How to use a standalone Expo module
 description: Learn how to use a standalone module created with create-expo-module in your project by using a monorepo or publishing the package to npm.
 ---
 
+import { Grid01Icon } from '@expo/styleguide-icons/outline/Grid01Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
 import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
 
-**The recommended way to create an Expo module** inside an existing application is described in the [Expo Modules API: Get started](/modules/get-started/) guide. This page outlines two additional methods of using a module created with `create-expo-module` inside an existing project:
+**The recommended way to create an Expo module** in an existing application is described in the [Expo Modules API: Get Started](/modules/get-started/) guide. This page explains two additional methods for using a module created with `create-expo-module` in an existing project:
```

Comment:
```suggestion
**The recommended way to create an Expo module** in an existing project is described in the [Expo Modules API: Get Started](/modules/get-started/) guide. This tutorial explains two additional methods for using a module created with `create-expo-module` in an existing project:
```

We always prefer to use the term "app" over "application" throughout the docs to reference an arbitrary app. However, in this case, the word project is more suitable since it references the app stage (development). We are also using the same word in the last line of this paragraph.

This page is also a tutorial so instead of using a generic term, let's be specific.